lib/validation.tsx

```diff
@@ -358,3 +358,33 @@ export const isSymbol = (field: string): boolean => {
 // Helpful to check whether a referer is local vs. an external URL
 // Note: a negated version of https://stackoverflow.com/questions/10687099/how-to-test-if-a-url-string-is-absolute-or-relative
 export const localPathRegEx = new RegExp("^(?!((?:[a-z+]+:)?//))", "i");
+
+/**
+ * This function tests whether a string contains a UUID
+ * @param field A string containing a UUID
+ * @returns {boolean} The validation result
+ */
+export const isUUID = (field: string): boolean => {
+  // Regex from: https://github.com/microsoft/uuid-validate/blob/06554db1b093aa6bb429156fa8964e1cde2b750c/index.js#L2
+  const reg = new RegExp(
+    "^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[0-9a-f]{4}-[0-9a-f]{12}$",
```

Comment:
The Regex I have used to validate the payload that will be passed to the API is `"^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$"`.
I noticed that the one you have uses a limitation for letters between A and F which appears to be correct. We could change the one I have in file `uuid-array.schema.json` as part of this pull request.

However I am not sure about the middle section of your expression `[0-9a-f]{4}-[1-5][0-9a-f]{3}-[0-9a-f]{4}`. It may be correct for UUID v5 but in our system we are using UUID v4.

Reply:
@thiessenp-cds Let me know what you think and if you want me to deal with the change in `uuid-array.schema.json `.

Reply:
Oh good point, thanks for pointing that out! I just assumed the latest UUID version was used. In that case, probably best to not use this microsoft schema.  I'll update the validation.tsx to use the same UUID schema in uuid-array.schema.json

Reply:
You may want to keep the limitation in range of letters though since it appears to be a valid one. From `A to Z` to `A to F`. You will have to also apply that change in the `uuid-array.schema.json` file

Reply:
Sounds good. Done
I also pointed the form id/name regex to point to the schema validation one - better to have the regex in one place

Reply:
Perfect! Thank you :)